.circleci/config.yml

```diff
@@ -190,6 +190,7 @@ commands:
             echo 'export FEATURE_BUSINESS_FLC=$FEATURE_BUSINESS_FLC_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FCORP=$FEATURE_BUSINESS_FCORP_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FNP=$FEATURE_BUSINESS_FNP_DEV' >> $BASH_ENV
+            echo 'export FEATURE_BUSINESS_NP=$FEATURE_BUSINESS_NP_DEV' >> $BASH_ENV
```

Comment:
[sand - fix this as a follow-up commit]

In order for the `feature-flags` page to work, we need to pass ALL environment variables into the dev env so that it can display their values.

So *in addition to* this line, which is good as is, we need to set the STAGING and PROD env vars here in the dev section. You can see this happening below on lines 195-198 and lines 200-203. We need to do this for both NP and FNP. Note that the "live" variable on this line here gets renamed without the environment tag (ie, `FEATURE_BUSINESS_NP_DEV` becomes `FEATURE_BUSINESS_NP_DEV`) but for the additional staging/prod variables, it keeps its original name with the `_STAGING` or `_PROD`